src/i18n.ts

```diff
@@ -4,9 +4,11 @@ import { initReactI18next } from 'react-i18next';
 import en from './i18n/locales/en.json';
 import fr from './i18n/locales/fr.json';
 
+import Cookies from "js-cookie";
+
 i18n.use(initReactI18next).init({
   fallbackLng: 'en',
-  lng: 'en',
+  lng: Cookies.get("lang_setting"),
```

Comment:
id make sure we have a default value for `en` here as the cookie might not already exists.

Reply:
the fallbackLng is there in case the Cookie does not exist. It is considered the backup lang in case there's no override.

Reply:
Appologies!